tests/tasks/test_clone.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright (c) 2021 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import os
+
+from atomic_reactor import source
+from atomic_reactor.tasks import common
+from atomic_reactor.tasks import clone
+from tests.constants import DOCKERFILE_GIT
+
+
+class TestCloneTask:
+    """Tests for the CloneTask class."""
+
+    def test_clone_execute(self, tmpdir):
```

Comment:
We have an `@requires_internet` decorator for tests that require internet access. Alternatively, this test could just mock the `get()` method and assert that it was called, the method has its own unit tests